.github/workflows/OntologyMapper_Prod.yml

```diff
@@ -0,0 +1,56 @@
+name: Build, Package, and Push to Feed
+
+on:
+  push:
+    branches: [ "master" ]
+
+env: 
+  AZURE_ARTIFACTS_FEED_URL: https://pkgs.dev.azure.com/dynamicscrm/Solutions/_packaging/Solutions-SmartPlacesAndEnergy/nuget/v3/index.json    
+  BUILD_CONFIGURATION: 'Release'    # set this to the appropriate build configuration
+  WORKING_DIRECTORY: OntologyMapper
+  DOTNET_VERSION: '6.x' 
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: Setup .NET
+      uses: actions/setup-dotnet@v2
+      with:
+        dotnet-version: 6.0.x
+    - name: Restore dependencies
+      run: dotnet restore "${{ env.WORKING_DIRECTORY }}"
+    - name: Build
+      run: dotnet build "${{ env.WORKING_DIRECTORY }}" --no-restore
+    - name: Test
+      run: dotnet test "${{ env.WORKING_DIRECTORY }}" --no-build --verbosity normal
+
+  az-artifacts-build-and-deploy:
+    needs: build
+    runs-on: ubuntu-latest
+    steps:
+      # Checkout the repo
+      - uses: actions/checkout@v2
+
+      # Setup .NET Core SDK
+      - name: Setup .NET Core
+        uses: actions/setup-dotnet@v1
+        with:
+          dotnet-version: ${{ env.DOTNET_VERSION }}
+          source-url: ${{ env.AZURE_ARTIFACTS_FEED_URL }}
+        env:
+          NUGET_AUTH_TOKEN: ${{ secrets.AZURE_ARTIFACTS_PAT }} 
+
+      # Run dotnet build and package
+      - name: dotnet build and publish
```

Comment:
Why is this doing this again? the build from line 14 already did this?

Reply:
Fixing